go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/containerd/typeurl/v2 v2.1.1
 	github.com/creack/pty v1.1.18
 	github.com/distribution/reference v0.5.0
-	github.com/docker/cli v26.0.0-rc1+incompatible
+	github.com/docker/cli v26.0.0+incompatible
 	github.com/docker/cli-docs-tool v0.7.0
 	github.com/docker/docker v26.0.0-rc1+incompatible
```

Comment:
oops merged too quickly :see_no_evil:, forgot to bump https://github.com/docker/buildx/pull/2359/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R19